backend/resolvers/queries/scholarshipQuery.js

```diff
@@ -0,0 +1,18 @@
+import { UserError } from "graphql-errors";
+import { ADMIN } from "../../constants";
+import Scholarship from "../../models/scholarship";
+
+export function scholarship(_, args, req) {
+  if (
+    req.auth.type !== ADMIN
+  ) {
+    throw new UserError("Permission Denied");
+  }
+
+  // Apply ordering, if desired
+  const order = args.orderBy
+    ? { order: [[args.orderBy.sort, args.orderBy.direction]] }
```

Comment:
Is there a test that does tests out the sorting?

Reply:
Not one directly. I did a quick manual test and it worked okay.